org.thingpedia.iot.security-camera/manifest.tt

```diff
@@ -1,22 +1,31 @@
-class @com.nest.security_camera extends @security-camera {
-  import loader from @org.thingpedia.embedded();
-  import config from @org.thingpedia.config.builtin();
+abstract class @org.thingpedia.iot.security-camera
+#_[thingpedia_name="Security Camera and Related Sensors"]
+#_[thingpedia_description="Interface for Security Camera and Related Sensors."]
+#_[description="Interface for Security Camera and Related Sensors."]
+#[license="BSD-3-Clause"]
+#[license_gplcompatible=true]
+#[subcategory="home"]
+{
+  monitorable query state(out state : Enum(detecting,not_detecting))
+  #_[canonical="sensor state"]
+  #_[confirmation="the state of $__device"]
+  #_[formatted=["The sensor is ${state:enum} movement."]];
```

Comment:
What is this? This function does not make sense. The security camera interface should only have security camera features, and motion sensor features should go into the motion sensor interface. `com.nest.security_camera` can implement all of `org.thingpedia.iot.security-camera`, `org.thingpedia.iot.motion`, `org.thingpedia.iot.sound` and `org.thingpedia.iot.occupancy`.

Reply:
Dropped org.thingpedia.iot.security-camera for now